### ISSUE TITLE
Event yearChanged is not mentioned in documentation

### ISSUE THREAD

**Issue by DarkSide666** [Paul-DS/bootstrap-year-calendar#106]
Event yearChanged is not mentioned in documentation: http://www.bootstrap-year-calendar.com/#Documentation/Events

**Comment by Paul-DS** [comment]
Yes, my bad.

I wanted to keep the documentation coherent with the latest release.
So the documentation you can find on the website is for the v1.1.0, not the current version in development.
I should push a new release, and maybe modified the website to allow choosing the version of the documentation (or at least include the current docs in the repo)

**Comment by DarkSide666** [comment]
BTW, you can check out https://readthedocs.org/. You can set up documentation there.
Here is one project which use it: http://agile-data.readthedocs.io and you can keep docs files directly in your github repo (like here https://github.com/atk4/data/tree/develop/docs).
That way documentation would always be linked with actual source code version you have in github.

**Comment by Paul-DS** [comment]
Yes I think this is the solution I'll take. I'll update everything for the next release (hopefully soon)
